tests/execute/reboot/efi/test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Check reboot variables"
+        for variable in TMT_REBOOT_COUNT RSTRNT_REBOOTCOUNT REBOOTCOUNT; do
+            rlLog "$variable=${!variable}"
+            rlRun "[[ -n '${!variable}' ]]" 0 \
+                "Reboot count variable '$variable' must be defined."
+        done
+    rlPhaseEnd
+
+    # Before
+    if [ "$TMT_REBOOT_COUNT" == "0" ]; then
+        rlPhaseStartTest "Before reboot"
+            if [ $(command -v efibootmgr) &>/dev/null ]; then
+                rlLog "efibootmgr installed."
+                rlRun "current=$(efibootmgr | awk '/BootCurrent/ { print $2 }')" 0 "Ascertain BootCurrent value."
+                rlLog "BootCurrent=$current"
+            fi
+            rlRun "tmt-reboot" 0 "Reboot using 'tmt-reboot'."
+            # Add sleep to check that the test is killed by tmt-reboot
+            rlRun "sleep 3600"
+        rlPhaseEnd
+
+    # First
+    elif [ "$TMT_REBOOT_COUNT" == "1" ]; then
+        rlPhaseStartTest "After first reboot"
+            rlRun "syslog=/var/log/messages"
+            rlRun "count=$(grep \"efibootmgr -n $current\" $syslog | wc -l)"
```

Comment:
I believe that after the first reboot the `$current` variable will not be set because the `if` branch above will be skipped completely. How is the `grep` check supposed to work? Or have I missed anything?

Reply:
Indeed. I've updated the logic to output the value to a file, for reading after the machine boots